std/didexchange/models.go

```diff
@@ -28,7 +28,7 @@ type Response struct {
 	ConnectionSignature *ConnectionSignature `json:"connection~sig,omitempty"`
 	Thread              *decorator.Thread    `json:"~thread,omitempty"`
 
-	connection *Connection // Actual data, to be signed or verified
+	Connection *Connection `json:"-"` // Actual data, to be signed or verified
```

Comment:
I could not see here why this need to be exported, but I suppose there is a reason?

Reply:
It's needed in the coming PR 🙂 Dropped it out from json output so that there are no effect on protocol messages.